tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/compose/HelmUpgradeExtensionIT.java

```diff
@@ -110,11 +110,7 @@ void withBridgeConfiguration_enableDisableBridged() throws Exception {
         assertThat(hivemqCustomResource.get().getAdditionalProperties().get("spec").toString()).matches(
                 ".*extensions=\\[.*?enabled=false,.*?id=hivemq-bridge-extension,.*?].*");
 
-        HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME,
-                "--cascade",
-                "foreground",
-                "--namespace",
-                namespace);
+        HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME, namespace);
```

Comment:
Do we need the try-finally contruct here as well?

Reply:
Refactored some ITs to remove boilerplate code, so it also deletes the namespace inside the `uninstallRelease` method.